create_infotexts.py

```diff
@@ -367,8 +369,8 @@ def process_depicted_place(self, place_string, places_mapping):
                 place_as_wikitext = place_string
 
             # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-                if places_mapping[place_string].get('commonscat'):
-                    self.content_cats.append(places_mapping[place_string]["commonscat"])
+            if places_mapping[place_string].get('commonscat'):
```

Comment:
I would avoid making this change here since it is unrelated to what you want to do and will make this PR collide with the other patches.